src/Stories.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { storageKey, Box, SimpleGrid, Grid} from "@chakra-ui/react"
+import { Image, Heading } from "@chakra-ui/react"
+import { Button, ButtonGroup, Stack } from "@chakra-ui/react"
+import UserStory from "./UserStory";
+export default class Stories extends React.Component {
+    getStories(){
+        let Storieses = [];
+        for (let i = 0; i < 10; ++i) {
+            Storieses.push(<UserStory />);
+        }
+        console.log(Storieses);
```

Comment:
Perhaps it would be better to simply leave a comment about what this function should do.
```suggestion
    // TODO: Fetch stories from backend
```